docs/advanced-usage/set-up-natural-language-search.md

```diff
@@ -30,19 +27,28 @@ Uploading embeddings to Kolena can be done in three simple steps:
 Let's take a look at each step with example code snippets.
 
 ### Step 1: Install `kolena_embeddings` Package
-
-Copy the `kolena_embeddings-*.*.*.tar.gz` file (provided by your Kolena contact) to your working directory, and install it as a dependency.
+The package can be installed via `pip` or `poetry` and requires use of your kolena token which can be created
+on the [:kolena-developer-16: Developer](https://app.kolena.io/redirect/developer) page.
 
 === "`pip`"
 
     ```shell
-    pip install ./kolena_embeddings-*.*.*.tar.gz
+    pip install --extra-index-url="https://MY_KOLENA_TOKEN@gateway.kolena.cloud/repositories" kolena-embeddings
     ```
 
 === "`poetry`"
+    Configure an additional poetry source:
+    ```shell
+    poetry source add --priority=supplemental kolena-embeddings "https://gateway.kolena.cloud/repositories"
+    ```
+    Then run:
+    ```shell
+    poetry config http-basic.kolena-embeddings "KOLENA_TOKEN" ""
```

Comment:
What's our common practice to make it clear that this is NOT a verbatim "KOLENA_TOKEN" but something to replaced with `<KOLENA_TOKEN>`?

Reply:
I think calling it out like I did in the latest commit "making sure to replace <KOLENA_TOKEN> with the token retrieved from the developer page" makes it clear. Let me know if you have any suggestions.